src/fcl_model.cpp

```diff
@@ -89,7 +89,6 @@ void FCLModelTpl<DATATYPE>::init(urdf::ModelInterfaceSharedPtr const &urdfTree,
     urdf::LinkConstSharedPtr root_link = urdf_model->getRoot();
     dfs_parse_tree(root_link, "root's parent");
     auto tmp_user_link_names = collision_link_names;
-    std::sort(tmp_user_link_names.begin(), tmp_user_link_names.end());
```

Comment:
I removed this because it looked fishy. The user specified the order the links are connected but sorting can destroy this order. @Colin97 why was it put in in the first place?